sklearn/neighbors/approximate.py

```diff
@@ -217,6 +217,10 @@ def __init__(self, n_estimators=10, radius=1.0, n_candidates=50,
         self.min_hash_match = min_hash_match
         self.radius_cutoff_ratio = radius_cutoff_ratio
 
+        warnings.warn("LSHForest has poor performance.")
```

Comment:
Please remove that line. Or making an inline comment instead of a dedicated UserWarning.